zeppelindemo/resources/projects.xml

```diff
@@ -76,18 +76,71 @@ into the zeppelin-env.sh file. <br />
 <pre>
 $ sudo /opt/zeppelin/bin/zeppelin-daemon.sh start
 </pre>
-6. Configure Cassandra log appenders.<br />
+6. Downgrade Cassandra version, since Zeppelin compatible only with not higher than 2.2 (by default on Sandbox Cassandra version 3.5 is installed). <br />
+Add the DataStax Community repository to the /etc/apt/sources.list.d/cassandra.sources.list. <br />
+<pre>
+echo "deb http://debian.datastax.com/community stable main" | sudo tee -a /etc/apt/sources.list.d/cassandra.sources.list
+curl -L http://debian.datastax.com/debian/repo_key | sudo apt-key add -
+</pre>
+Install Cassandra 2.2.5. <br />
+<pre>
+sudo apt-get update
+sudo apt-get install cassandra=2.2.5
+</pre>
+While installation will be following question, use default answer to avoid compatibility issues. <br />
+<pre>
+Configuration file '/etc/cassandra/cassandra.yaml'
+ ==> Modified (by you or by a script) since installation.
+ ==> Package distributor has shipped an updated version.
+   What would you like to do about it ?  Your options are:
+    Y or I  : install the package maintainer's version
+    N or O  : keep your currently-installed version
+      D     : show the differences between the versions
+      Z     : start a shell to examine the situation
+ The default action is to keep your current version.
+*** cassandra.yaml (Y/I/N/O/D/Z) [default=N] ? 
+N
+</pre>
+You can check if the Cassandra service is running by executing the following command. <br />
+<pre>
+sudo netstat -ntlp | grep 9042
+</pre>
+Install Java Native Access (JNA). <br />
+<pre>
+sudo apt-get install libjna-java
+</pre>
+Reboot Cassandra cluster. <br />
+<pre>
+sudo cassandra
+</pre>
+Check cassandra cql shell. <br />
+<pre>
+cqlsh
+Connected to Test Cluster at 127.0.0.1:9042.
+[cqlsh 5.0.1 | Cassandra 2.2.5 | CQL spec 3.3.1 | Native protocol v4]
+Use HELP for help.
+cqlsh>
+</pre>
+8. Add initial data into Cassandra cluster from "cqlsh" shell. <br />
+<pre>
+CREATE KEYSPACE logs WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
+use logs;
+CREATE OR REPLACE FUNCTION logs.timeFormat(time bigint) RETURNS NULL ON NULL INPUT RETURNS text LANGUAGE java AS $$ return new java.text.SimpleDateFormat("mm:ss").format(new java.util.Date(time));$$;
+CREATE OR REPLACE FUNCTION logs.timeDelta(arg bigint, delta bigint) RETURNS NULL ON NULL INPUT RETURNS bigint LANGUAGE java AS $$ return arg - delta;$$;
+</pre>
+9. Recover tables <b>"panel_per_row"</b> and <b>"zone_per_row"</b> in <b>"logs"</b> keyspace. 
+It will be created automatically, if you go to Kaa Admin UI and do empty-edit (just edit name and revert back) of both according appenders and click <b>Save</b> button. <br />
+10. Configure Cassandra log appenders.<br />
 Navigate to the Kaa server administrative interface by clicking the "Administrative console" link at the top of the Sandbox window.
```

Comment:
The name of the Administrative UI has been changed recently. Please use the new (correct) naming in the documentation.